addons/website/static/src/builder/plugins/form/form_option.xml

```diff
@@ -231,6 +231,60 @@
                 records="state.valueList.availableRecords" />
         </div>
     </BuilderRow>
+    <BuilderRow label.translate="Character Limit" t-if="domStateCurrentFieldInput.isTextArea or ['text'].includes(domStateCurrentFieldInput.type)">
+        <BuilderCheckbox
+            id="'character_limit_opt'"
+            dataAttributeAction="'characterLimit'"
+            dataAttributeActionValue="'true'"
+            action="'toggleCharacterLimit'"
+            preview="false"
+            applyTo="`input, textarea`"
+        />
+    </BuilderRow>
+    <t t-if="isActiveItem('character_limit_opt')">
+        <BuilderRow label.translate="Min Characters" level="1">
+            <BuilderNumberInput
```

Comment:
<img width="284" height="100" alt="image" src="https://github.com/user-attachments/assets/54504e77-d3ef-42d9-9e9f-d42b18c88b06" />

I can enter min value more than max value, shouldn't we add validation in Builder as well?